package.json

```diff
@@ -27,7 +27,8 @@
     "precompile": "gts clean"
   },
   "dependencies": {
-    "google-gax": "^3.5.2"
+    "google-gax": "^3.5.2",
+    "@google-cloud/bigquery": "^6.1.0"
```

Comment:
meta-issue:  does storage depending on bigquery core mean we can't ever invert this in the main nodejs-bigquery library for read acceleration (circular dependencies)?  Does adapt need to live elsewhere?

Reply:
I removed the dependency on nodejs-bigquery and copied just the basic definition needed for the conversion. Was only being used for typings in the end.